src/mono/wasm/runtime/logging.ts

```diff
@@ -91,8 +91,8 @@ export function mono_wasm_symbolicate_string(message: string): string {
 
 export function mono_wasm_stringify_as_error_with_stack(err: Error | string): string {
     let errObj: any = err;
-    if (!errObj || !errObj.stack || !(errObj instanceof Error)) {
-        errObj = new Error(errObj || "Unknown error");
+    if (!errObj || !errObj.stack) {
+        errObj = new Error(errObj ? ("" + errObj) : "Unknown error");
```

Comment:
Why do we need this ""? Is it just for conversion to string?
```suggestion
        errObj = new Error(errObj ? `${errObj}` : "Unknown error");
```

Reply:
this is just conversion to string, it's cheaper than the interpolation, I think.